### DISCUSSION TITLE
Error when trying to migrate Gatsby App from "netlify-cms-app" to "decap-cms-app"

### DISCUSSION THREAD

**Discussion by praharsh23** [decaporg/decap-cms#6911] [Q&A]
Hi all,

I'm receiving an error when trying to migrate from "netlify-cms-app" to "decap-cms-app".

Any suggestions to resolve this?

Also, is there any doc to help with migration of existing apps that use `netlify-cms` to `decap-cms`?

**Steps taken:**

1) npm uninstall netlify-cms-app
2) npm install decap-cms-app
3) In cms.js of my App changed the import as follows

`import CMS from 'decap-cms-app'` 
instead of 
`import CMS from 'netlify-cms-app'`

Here is the error message with `gatsby build`

```
 ERROR #11321  PLUGIN

"gatsby-plugin-netlify-cms" threw an error while running the onCreateWebpackConfig lifecycle:

Cannot find module 'netlify-cms-app\dist\netlify-cms-app.js'
```

**Comment by martinjagodic** [comment]
This seems to be a problem with `gatsby-plugin-netlify-cms`. The package is maintained by Gatsby, so I suggest that you open an issue with them.

**Reply by praharsh23** [reply]
Thanks for the response. Started a discussion in the Gatsby forum here: https://github.com/gatsbyjs/gatsby/discussions/38546
